index.html

```diff
@@ -11,12 +11,18 @@
       <header>
         <h1 id = "title">Freecodecamp Survey Form</h1>
         <p id = "description">Thank you for taking the time to help us improve the platform</p>
+        <!-- <nav>
+          <ul>
+            <li>Home</li>
+            <li>Contact</li>
+          </ul>
+        </nav> -->
       </header>
     
     <form id = "survey-form">
       <div>
         <label id ="name-label"for="">Name</label><br>
-        <input id = "name" type="text" required="required" placeholder="Enter your name " />
+        <input id = "name-label" type="text" required="required" placeholder="Enter your name " />
```

Comment:
Duplicate id kindly makes the id unique , it should not be same . :+1:

Reply:
I suggest you try as much as possible to communicate clearly to not confuse the student. 
Also, as previously requested, try to always state the 'why' a change is required in order to help the student learn better.

instead of `Duplicate id kindly makes the id unique, it should not be same ` say `I suggest you use a different Id for these elements as this will make the unique and will make styling them very easier using their Id since an Id can not be used multiple times`